src/features/search/Search.tsx

```diff
@@ -126,27 +148,54 @@ function Search() {
               <Alert variant="warning">{state.crawl.error.message}</Alert>
             </>
           )}
-          <Form.Check
-            value="Blackrock"
-            checked={realm === "Blackrock"}
-            type={"radio"}
-            id={"blackrock-radio"}
-            label={"Blackrock"}
-            onChange={handleRealmChange}
-          />
-          <Form.Check
-            value="icecrown"
-            checked={realm === "icecrown"}
-            type={"radio"}
-            id={`icecrown-radio`}
-            label={"Iceclown"}
-            onChange={handleRealmChange}
-          />
-        </Col>
-        <Col lg="2">
-          <Button variant="primary" type="submit">
-            Search
-          </Button>
+          <Row className="d-flex justify-content-center">
+            <Col className="d-flex justify-content-center">
+              <div className="radio-container">
+                <input
+                  className="custom-form-check"
+                  type="radio"
+                  id="blackrock-radio"
+                  value="Blackrock"
+                  checked={realm === "Blackrock"}
+                  onChange={handleRealmChange}
+                />
+                <label
+                  htmlFor="blackrock-radio"
+                  className={`radio-label blackrock${
+                    realm === "Icecrown" ? " anti-selected-realm" : ""
+                  }${hoveredRealm === "Icecrown" ? " anti-hovered-realm" : ""}`}
+                  onMouseEnter={() => setHoveredRealm("Blackrock")}
+                  onMouseLeave={() => setHoveredRealm(null)}
+                >
+                  Blackrock
+                </label>
+              </div>
+            </Col>
+            <Col className="d-flex justify-content-center">
+              <div className="radio-container">
+                <input
+                  className="custom-form-check"
+                  type="radio"
+                  id="icecrown-radio"
+                  value="Icecrown"
```

Comment:
```suggestion
                  value="Iceclown"
```
realms === servers

On warmane there are technically three but >90% play on either Blackrock or Icecrown. The big difference is that Blackrock is PVP only. Icecrown gives you the ability to do quests and raids and shit. A common insult to people who play on Icecrown is IceCLOWN